test/tools/ossfuzz/config/solidity.dict

```diff
@@ -0,0 +1,214 @@
+" address(this).balance "
+" address(this).call("
+" address(this).callcode("
+" address(this).delegatecall("
+" address(this).send("
+" address(this).transfer("
+" block.blockhash "
+" block.coinbase "
+" block.difficulty "
+" block.gaslimit "
+" block.number "
+" block.timestamp "
+" days"
+" ether"
+" finney"
+" gasleft() "
+" hours"
+" keccak256.gas() "
+" keccak256.value() "
+" minutes"
+" msg.data "
+" msg.gas "
+" msg.sender "
+" msg.sig "
+" msg.value "
+" now"
+" ripemd160.gas() "
+" ripemd160.value() "
+" seconds "
+" sha256.gas() "
+" sha256.value() "
+" szabo "
+" tx.gasprice "
+" tx.origin "
+" weeks "
+" wei "
+" years "
+"!="
+"%"
+"&"
+"("
+")"
+"*"
+"**"
+"+"
+"++"
+"-"
+"--"
+".balance "
+".call("
+".callcode("
+".creationCode"
+".delegatecall("
+".gas();"
+".kill("
+".length "
+".pop();"
+".push("
+".runtimeCode"
+".send("
+".staticcall("
+".transfer("
+".value "
+"/"
+"//"
+"0**0"
+"1.1"
+"2e10"
+":="
+";"
+"<"
+"<<"
+"<="
+"=="
+">"
+">="
+">>"
+"[a, b, c]"
+"\\udead"
+"\\xff"
+"^"
+"abi.encode("
+"abi.encodePacked("
+"abi.encodeWithSelector("
+"abi.encodeWithSignature("
+"add("
+"addmod("
+"anonymous"
+"assembly {"
+"assert("
+"bool "
+"break;"
+"byte "
+"bytes("
+"bytes1 "
+"bytes10 "
+"bytes11 "
+"bytes12 "
+"bytes13 "
+"bytes14 "
+"bytes15 "
+"bytes16 "
+"bytes17 "
+"bytes18 "
+"bytes19 "
+"bytes2 "
+"bytes20 "
+"bytes21 "
+"bytes22 "
+"bytes23 "
+"bytes24 "
+"bytes25 "
+"bytes26 "
+"bytes27 "
+"bytes28 "
+"bytes29 "
+"bytes3 "
+"bytes30 "
+"bytes32 "
+"bytes4 "
+"bytes5 "
+"bytes6 "
+"bytes7 "
+"bytes8 "
+"bytes9 "
+"console.log("
+"constant "
+"constructor "
+"continue;"
+"contract "
+"delete "
+"do"
+"ecrecover("
+"else"
+"emit a("
+"enum B {"
+"event e("
+"external "
+"false "
+"fixed "
+"fixed128x128 "
+"for (a=0;a<2;a++)"
+"function bid() public payable {"
+"hex\"001122FF\""
+"if"
+"int "
+"int x = -2**255;"
+"int256 "
+"int8 "
+"interface i {"
+"internal "
+"is"
+"keccak256("
+"let x := "
+"library l {"
+"log0("
+"log1("
+"log2("
+"log3("
+"log4("
+"mapping("
+"memory m ="
+"modifier onlySeller() {"
+"mulmod("
+"new "
+"payable "
+"pragma experimental ABIEncoderV2;"
+"pragma experimental SMTChecker;"
+"pragma solidity >=0.4.0;"
+"pragma solidity ^90.90.0"
+"public "
+"pure "
+"require("
+"require(msg.sender == 0,\"\""
```

Comment:
Perhaps better to remove the `\"\"` since we also want the type checker to go into edge cases where the types are not as expected.

Reply:
I think the `"require("` will take care of synthesizing type mismatched constructions.

Reply:
ah ok!